### PR TITLE
Use multi-line INSERT statements to speed up regression tests

### PR DESCRIPTION
**Pull request by hlinnaka**

### CODE REVIEW

**Comment by jimmyyih** [comment]
+1 from me

File: src/test/regress/bugbuster/expected/aggfunc.out [hunk 1]
```diff
@@ -40,143 +40,126 @@ select count(*) from testemp;
...
+create table quantity (qty integer, price integer, product character);
 NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 'qty' as the Greenplum Database data distribution key for this table.
 HINT:  The 'DISTRIBUTED BY' clause determines the distribution of data. Make sure column(s) chosen are the optimal data distribution key to minimize skew.
-
--- inserting values into quantity table
-insert into quantity values (100, 50, p1);
+-- inserting values into quantity table
+insert into quantity values (100, 50, p1);
 ERROR:  column "p1" does not exist
 LINE 1: insert into quantity values (100, 50, p1);
```
Comment (asimrp) [inline src/test/regress/bugbuster/expected/aggfunc.out]: Not sure why we are deliberately inserting incorrect values here... I understand that this is not related to the current change but thought of noting the question still.

Comment (hlinnaka) [inline src/test/regress/bugbuster/expected/aggfunc.out]: On 06/12/15 09:25, Asim RP wrote:
>>   NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 'qty' as the Greenplum Database data distribution key for this table.
>>   HINT:  The 'DISTRIBUTED BY' clause determines the distribution of data. Make sure column(s) chosen are the optimal data distribution key to minimize skew.
>> -
>> --- inserting values into quantity table
>> -insert into quantity values (100, 50, p1);
>> +-- inserting values into quantity table
>> +insert into quantity values (100, 50, p1);
>>   ERROR:  column "p1" does not exist
>>   LINE 1: insert into quantity values (100, 50, p1);
>
> Not sure why we are deliberately inserting incorrect values here... I
> understand that this is not related to the current change but thought
> of noting the question still.

Ah yeah, I saw those too and wondered the same. But as you said, it's a 
separate issue.

- Heikki

**Comment by hlinnaka** [comment]
Pushed, thanks for the review!

**Pull request closed without merging**